dev/com.ibm.ws.beanvalidation.v20_fat/test-applications/bvalCDIApp/src/bval/v20/cdi/web/BeanValCDIServlet.java

```diff
@@ -42,6 +42,7 @@ public void testInjectVal() throws Exception {
         Assert.assertNotNull(injectVal);
     }
 
+    // TODO: Disabled because bean is not being injected into TestAnnotationValidator
```

Comment:
can undo changes to this file

Reply:
Change due to basing off of feature branch and then switching to master.